src/CSharp/EasyMicroservice.FileManager/Extensions/DirectoryExtensions.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using EasyMicroservice.FileManager.Models;
 using System.Threading.Tasks;
 
-namespace EasyMicroservice.FileManager
+namespace EasyMicroservice.FileManager.Extensions
```

Comment:
For the 'Extensions' We don't use the 'extensions' namespace because of developer usage will be harder.